examples/client/Locomotion/src/pages/RideHistory/RangeDateTimePicker.js

```diff
@@ -22,7 +24,7 @@ export default ({
   return (
     <>
       <DatePicker
-        textColor={theme.textColor}
+        textColor={colorScheme === 'dark' ? '#ffffff' : theme.textColor}
```

Comment:
you can create a generic function to handle these, its the same condition and resutl